pkg/asset/installconfig/filefetcher_generated_test.go

```diff
@@ -0,0 +1,60 @@
+// Code generated by MockGen. DO NOT EDIT.
```

Comment:
Also this file is same as like `mock/file_fetcher_generated.go` ?

Reply:
Yes. This file should not be here. I originally created it here before moving it to `mock/file_fetcher_generated.go` to make it available to multiple packages.